### ISSUE TITLE
useEffect callbacks are not invoked for subsequent tests if there are more than one useEffect hooks

### ISSUE THREAD

**Issue by MrCreeper1008** [mikeborozdin/jest-react-hooks-shallow#7]
If a component has more than one useEffect hook, the callback will be invoked only for the very first test in a `describe` block.

A very simple reproduction:
```javascript
function Component() {
  useEffect(() => {
    console.log('test 1');
  }, []);

  useEffect(() => {
    console.log('test 2');
  }, []);

  return null;
}

describe('test', () => {
  it('useEffect works', () => {
    shallow(<Component />);
  });

  it('useEffect does not work', () => {
    shallow(<Component />);
  });
})
```

Note that if I remove t he deps array everything will be back to normal.

**Comment by mikeborozdin** [comment]
Hi @MrCreeper1008 ,

Thanks a lot for reporting this. I'll look into it.

**Comment by mikeborozdin** [comment]
@MrCreeper1008 ,

Which version are you using? Are you on the latest `1.2.0`? I've used your example - and I go get outputs in both cases:

```
test 1
test 2

test 1
test 2
```

**Comment by MrCreeper1008** [comment]
I'm on `1.1.0`

**Comment by mikeborozdin** [comment]
@MrCreeper1008 Can you please upgrade to `1.2.0`? It was fixed there, whereas `1.1.0` does have that issue.

**Comment by MrCreeper1008** [comment]
ok I'll give it a try. Thank you!
